web/Makefile

```diff
@@ -20,7 +20,7 @@ SUFFIXES:
 blob/files.go: $(shell find templates/ static/ -type f)
 	# Note that embed-static.sh excludes map files and the
 	# non-minified bootstrap files.
-	../scripts/embed-static.sh static templates | $(GOFMT) > $@
+	go generate ./blob
```

Comment:
Let's use `$(GO)` instead of `go` here to use the same Go environment as in the main Makefile.